### ISSUE TITLE
Fira website MVP

### ISSUE THREAD

**Issue by m3tablock** [opencorelabs/fira#42]
MVP landing page for website

[Figma link](https://www.figma.com/file/A3ojztItj6zZhnsq6axSF2/Fira?type=design&node-id=20-60&t=h3Cmrf4ScUyOl2h1-4)

![Image](https://github.com/opencorelabs/fira/assets/100637767/953d3359-532b-4ca5-9800-14e190afe509)

**Comment by pnwx** [comment]
I think for the earliest of versions we won't have register/MVP publicly enabled on the hosted instance - will direct people to self-host.

**Comment by harryhexhash** [comment]
Makes sense. I think that answers my other question about where the marketing site's code should live and I think it should be a separate client from the main app. 

Do we want it to be bundled/deployed with the main app or should we plan on hosting it elsewhere?

**Comment by m3tablock** [comment]
@harryhexhash re-opening this for now while we track towards final version

Looks great

+1 on @pnwx 's comments re self hosting to start, we won't have a hosted version to sign up for 

Updated logos are here: https://www.figma.com/file/A3ojztItj6zZhnsq6axSF2/Fira-(Public)?type=design&node-id=33%3A158&t=KgEmMPSPop242sI0-1

Let me know if you can export these as SVG, if not I can send them to you

**Comment by harryhexhash** [comment]
Sounds good. I will update the logos this week. Thanks!
